src/test/java/org/sagebionetworks/bridge/models/schedules2/timelines/ScheduledSessionTest.java

```diff
@@ -116,5 +117,25 @@ public void copyWithoutAssessments() {
         assertEquals(schSession.getTimeWindow(), window);
         assertTrue(schSession.isPersistent());
         assertTrue(schSession.getAssessments().isEmpty());
+        assertNull(schSession.getStudyBurstId());
+        assertNull(schSession.getStudyBurstNum());
+    }
+    
+    @Test
+    public void reportsStudyBurstInformation() {
+        Session session = new Session();
+        session.setGuid(SESSION_GUID_1);
+        
+        TimeWindow window = new TimeWindow();
+        window.setGuid(SESSION_WINDOW_GUID_1);
+        
+        ScheduledSession schSession = new ScheduledSession.Builder()
+                .withSession(session)
+                .withTimeWindow(window)
+                .withStartEventId("study_burst:foo:01")
+                .build();
```

Comment:
Does this test what you think it tests?

Reply:
Yes, this tests that the builder correctly parses that event ID into studyBurstId and studyBurstNum.